README.md

```diff
@@ -1,27 +1,27 @@
 NodeJS for OpenShift - Docker images
 ========================================
 
-This repository contains the sources and
-[Dockerfiles](https://github.com/openshift/sti-nodejs/tree/master/0.10)
-of the base images for deploying various versions of NodeJS applications as reproducible Docker
-images. The resulting images can be run either by [Docker](http://docker.io)
-or using [STI](https://github.com/openshift/source-to-image). 
-User can choose between RHEL7 and CentOS7 base image.
+This repository contains sources of the images for building various versions
+of NodeJS applications as reproducible Docker images using
+[source-to-image](https://github.com/openshift/source-to-image).
+User can choose between RHEL7 and CentOS7 based builder images.
+The resulting image can be run using [Docker](http://docker.io).
 
 
 Versions
 ---------------
-The versions we currently support in this repository are:
+The versions currently supported are:
 * **nodejs-0.10**
 
 
 Installation
 ---------------
-To build NodeJS image, choose between CentOS7 or RHEL7 base image:
-*  **RHEL7 base image**
+To build NodeJS image, choose between CentOS7 or RHEL7 based image:
+*  **RHEL7 based image**
```

Comment:
Scratch that ... I guess that it makes kinda sense to keep the OS version there (for now)